docs/remote/containers.md

```diff
@@ -23,7 +23,7 @@ This lets VS Code provide a **local-quality development experience** — includi
 
 **Local:**
 
-* **Windows:** [Docker Desktop](https://www.docker.com/products/docker-desktop) 2.0+ on Windows 10 Pro/Enterprise. (Docker Toolbox is not supported. Windows container images are not supported.)
+* **Windows:** [Docker Desktop](https://www.docker.com/products/docker-desktop) 2.0+ on Windows 10 Pro/Enterprise. Windows 10 Home (2004+) requires Docker Desktop 2.2+ and the [WSL2 back-end](https://aka.ms/vscode-remote/containers/docker-wsl2). (Docker Toolbox is not supported. Windows container images are not supported.)
```

Comment:
I had issues with Docker Desktop 2.2 and the WSL2 backend and ended up using 2.3 Edge. We have word that this will be released as stable sometime next week, so maybe say 2.3+ here for now.

Reply:
Yeah sounds good. I'd seen some chatter about that too.